app/javascript/src/components/home-title.vue

```diff
@@ -0,0 +1,97 @@
+<template>  
+  <div class="home-title">
+    <div class="home-title__title">
+      Busquemos un regalo para
+      <span class="home-title__user-name">{{ receiverName }}</span>
+      <img
+          class="home-title_icon home-title__icon--clear"
+          src="../assets/close-badge.svg"
+          @click="clearCookies"
+        >
+    </div>
+    <div class="home-title__subtitle">
+      Con el 
+      <img
+        class="home-title__icon home-title__icon"
+        src="../assets/gift-color-badge.svg"
+      >
+      las sugerencias mejorarán
+      <img
+        class="home-title__icon home-title__icon"
+        src="../assets/magic-wand.svg"
+      >
+      <span class="home-title__emphasis-word">
+        mágicamente
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+  import { mapState } from 'vuex';
+  export default {
+    name: "homeTitle",
+    computed: {
+      ...mapState([
+        'receiverName',
+      ]),
+    },
```

Comment:
Expected indentation of 2 spaces but found 4                               indent